### DISCUSSION TITLE
Is it possible to view pod logs?

### DISCUSSION THREAD

**Discussion by alejandromiller21** [kubeapps/kubeapps#6242] [Ideas]
Is it possible to view the pod logs of each helm deployment?

**Comment by absoludity** [comment]
Through the Kubeapps UX? No, not currently. We have wanted to add log viewing to Kubeapps, but it's never made it as a priority (we focus on enabling users to select apps from a catalog and install them, rather than managing resources and failures on the cluster). It was mentioned as part of #1524 but that was closed before adding the logs. It would be nice to add just as a basic view into a deployment failure... feel free to create a specific issue (or turn this into an issue), but we're low on resources at the moment so can't promise we'll get to it soon.
